src/rating.css.ts

```diff
@@ -0,0 +1,122 @@
+const minContainerSize = {
+    pointer: `calc(${minInteractiveArea.pointer} - ${iconSpacing})`,
+    touchable: `calc(${minInteractiveArea.touchable} - ${iconSpacing})`,
+};
+
+export const IconWrapper = style([
+    sprinkles({
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+    }),
+    {
+        // set width and height in order to ignore inline svg extra spacing
+        width: iconSize,
+        height: iconSize,
+
+        minWidth: minContainerSize.pointer,
+        minHeight: minContainerSize.pointer,
+        '@media': {
+            [mq.touchableOnly]: {
+                minWidth: minContainerSize.touchable,
+                minHeight: minContainerSize.touchable,
+            },
+        },
+    },
+]);
+
+export const halfIconContainer = sprinkles({
+    position: 'relative',
+});
+
+export const halfIconInactive = sprinkles({
+    position: 'absolute',
+});
+
+export const halfIconActive = style([
+    sprinkles({
+        position: 'relative',
+    }),
+    {
+        clipPath: 'polygon(0 0, 50% 0, 50% 100%, 0 100%)',
+        WebkitClipPath: 'polygon(0 0, 50% 0, 50% 100%, 0 100%)',
+    },
+]);
+
+export const disabled = style({
+    opacity: 0.5,
+});
+
+export const firstIcon = style({});
+export const lastIcon = style({});
+
+export const touchable = style({
+    vars: {
+        [interactiveAreaSize]: minInteractiveArea.pointer,
+    },
+    '@media': {
+        [mq.touchableOnly]: {
+            vars: {
+                [interactiveAreaSize]: minInteractiveArea.touchable,
+            },
+        },
+    },
+
+    transition: 'transform 0.4s cubic-bezier(0.175, 0.885, 0.32, 1.275)',
+    position: 'relative',
+
+    '::after': {
+        content: '',
+        position: 'absolute',
+        /**
+         * min() is not supported in old browsers (https://caniuse.com/css-math-functions).
+         * We don't force the minimum touchable area in that case.
+         */
+        top: [0, `min(0px, calc((100% - ${interactiveAreaSize}) / 2))`],
+        bottom: [0, `min(0px, calc((100% - ${interactiveAreaSize}) / 2))`],
+        left: [0, `min(0px, calc((100% - ${interactiveAreaSize}) / 2))`],
+        right: [0, `min(0px, calc((100% - ${interactiveAreaSize}) / 2))`],
+    },
+
+    selectors: {
+        [`&:not(${disabled}):active`]: {
+            transform: 'scale(1.2)',
+        },
+
+        // Add extra width on the left/right of interactive area so that we cover the space between icons
+        [`&:not(${firstIcon}):after`]: {
+            left: [
+                `calc(-1 * ${iconSpacing} / 2)`,
+                `min(calc(-1 * ${iconSpacing} / 2), calc((100% - ${interactiveAreaSize}) / 2))`,
+            ],
+        },
+        [`&:not(${lastIcon}):after`]: {
+            right: [
+                `calc(-1 * ${iconSpacing} / 2)`,
+                `min(calc(-1 * ${iconSpacing} / 2), calc((100% - ${interactiveAreaSize}) / 2))`,
```

Comment:
we should avoid using `min()` because we need[ Chrome >= 79](https://caniuse.com/?search=min), the min chrome version we support is 51

Reply:
That's why I've added a fallback. The same logic was used in IconButtons for example